blog/2023-01-18-introducing-netdata-paid-plans/index.md

```diff
@@ -0,0 +1,43 @@
+
+---
+
+Read more about Netdata introducing paid subscriptions. All Netdata functionality is and will be available for free in the Free Community Plan. Paid tiers include features targeted for businesses of different sizes and with different monitoring requirements.
+
+<!--truncate-->
+**Hello Netdata community**,
+
+We are excited to announce that we are introducing new paid subscriptions to our SaaS offering. At Netdata, we believe in providing free and unrestricted access to high-quality monitoring solutions, and our commitment to this principle will not change. We will continue to offer our free SaaS offering and Open Source Agent, which features unlimited nodes and users, unlimited metrics, and retention, providing real-time, high-fidelity, out-of-the-box infrastructure monitoring for packaged applications, containers, and operating systems.
+
+The new paid subscriptions are designed to provide additional features and capabilities for businesses that need tighter and customizable integration of the free monitoring solution to their processes, and they will be divided into three different plans: PRO, BUSINESS, and ENTERPRISE. Each plan will offer a different set of features and capabilities to meet the needs of businesses of different sizes and with different monitoring requirements.
+
+We understand that some of our users may have concerns about the introduction of paid subscriptions, so we would like to assure you that we will not be limiting access to features on the free plan. Instead, the free plan will continue to provide full access to all features available, and our paid subscriptions will allow you to control who can have access to which features.
+
+We also want to inform you that we are reviewing our Role-Based Access model to provide better User Administration capabilities, with the following new roles being added:
+
+- **Administrators** - Unrestricted access to all features and management of a Space.
+- **Managers** - Same as admins, but can't add nodes, or manage space settings (very similar to our original Member role)
+- **Troubleshooters** - Same as managers, but can’t manage users or rooms
+- **Observers** - Read only role, restricted to specific rooms.
+- **Billing** - Access to billing details and subscription management.
+
+The legacy Member role will be deprecated, and new users will have to be Administrators on the free COMMUNITY plan. 
+
+![image](./img/netdata-user-administration.png)
+
+We encourage you to visit our [pricing page](https://netdata.cloud/pricing) to learn more about the new plans and their features. If you have any questions or concerns, please reach out to our support team through [Email](mailto:info@netdata.cloud), [Github](https://github.com/netdata/netdata/discussions), the [Forums](https://community.netdata.cloud), and [Discord](https://discord.gg/2eduZdSeC7). We appreciate your continued support and are excited to offer these new paid subscriptions to our users.
+
```

Comment:
A screenshot of the new billing page would be cool to have as well. 

And a copy or callout to the FAQ as part of the blog.. so usual questions on am I charged per node, what if i have a lot of ephemeral nodes etc.. are answered right on the blog.